wolfssl/src/lib.rs

```diff
@@ -236,6 +236,16 @@ pub enum CurveGroup {
     /// `WOLFSSL_P521_KYBER_LEVEL5`
     #[cfg(feature = "postquantum")]
     P521KyberLevel5,
+
+    /// `WOLFSSL_P256_ML_KEM_512`
+    #[cfg(all(feature = "postquantum", not(feature = "kyber_only")))]
```

Comment:
`kyber_only` uses `postquantum` underneath.

`#[cfg(feature = "postquantum")]` should be enough

Reply:
I think if we use `#[cfg(feature = "postquantum")]`, the ML-KEM groups would still be included when we're using `kyber_only`, which is not we want when we are building a kyber-only client

Reply:
can I get an approval if this looks fine? @kp-mariappan-ramasamy